packages/@webex/plugin-meetings/test/integration/spec/journey.js

```diff
@@ -871,7 +871,7 @@ skipInNode(describe)('plugin-meetings', () => {
           assert.equal(bob.meeting.shareStatus, 'whiteboard_share_active');
         }));
 
-      it('alice adds chris as guest to 1:1 meeting', () =>
+      it('alice adds chris as guest to 1:1 meeting', async() =>
```

Comment:
why is this needed if there is no `await` in this function and it already returns a promise anyway (the result of Promise.all() call)